sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/student-loan-balances/_tests_/student-loan-balances-part-time-integration.scheduler.e2e-spec.ts

```
@@ -134,5 +138,26 @@ describe(
       // Expect student loan balance contains the student record.
       expect(studentLoanBalancesCount).toBe(0);
     });
+
+    it("Should throw an error that number of records mismatch when the records in the trailer does not match the number of records.", async () => {
```

"Should throw an error **that** number of records mismatch"